flink-runtime/src/test/java/org/apache/flink/runtime/net/SSLUtilsTest.java

```diff
@@ -432,21 +456,21 @@ public void testInvalidFingerprintParsing() throws Exception {
             SSLUtils.createInternalServerSSLEngineFactory(config);
             fail("expected exception");
```

Comment:
We'd better avoid using `fail` in the junit test now. Please also check the other place has same problem.

Reply:
Fixed, removed the fail method.